### PR TITLE
Kube-proxy not touching conntrack hashsize in containers. Use script …

### PR DESCRIPTION
**Pull request by ktsakalozos**
…to trigger pytest.

### CODE REVIEW

File: tests/test-distro.sh [hunk 1]
```diff
@@ -13,27 +13,37 @@ if echo "$*" | grep -q -- 'help'; then
     exit
 fi
 
-set -ue
+function create_machine() {
```
Comment (hyperbolic2346) [inline tests/test-distro.sh]: this makes me happy

File: tests/test-upgrade.py [hunk 2]
```diff
@@ -24,6 +24,13 @@ def test_upgrade(self):
         cmd = "sudo snap install microk8s --classic --channel={}".format(upgrade_from).split()
         check_call(cmd)
         wait_for_installation()
+        if is_container():
+            # In some setups (eg LXC on GCE) the hashsize nf_conntrack file under
+            # sys is marked as rw but any update on it is failing causing kube-proxy
+            # to fail.
```
Comment (hyperbolic2346) [inline tests/test-upgrade.py]: So we call path-kube-proxy, which will tell kube-proxy to not use conntrack.

**Comment by ktsakalozos** [comment]
Tests seem to be fine:
https://ci.kubernetes.juju.solutions/view/microk8s/job/build-and-release-microk8s-to-beta/49/console

File: tests/test-upgrade.py (hunk 2 above)
Comment (ktsakalozos) [reply tests/test-upgrade.py]: Easy

**Pull request merged**